posthog/cdp/validation.py

```diff
@@ -36,7 +36,8 @@ class InputsSchemaItemSerializer(serializers.Serializer):
     description = serializers.CharField(required=False)
     integration = serializers.CharField(required=False)
     integration_key = serializers.CharField(required=False)
-    integration_field = serializers.ChoiceField(choices=["slack_channel"], required=False)
+    required_field = serializers.CharField(required=False)
```

Comment:
Can we change this to `requires_field`? The current wording is confusing because it sounds like it makes this required.

Reply:
done - [`1212f16` (#26088)](https://github.com/PostHog/posthog/pull/26088/commits/1212f166e490a188dd8845e950413645c2861160)